types/node-telegram-bot-api/index.d.ts

```diff
@@ -647,6 +674,8 @@ declare namespace TelegramBot {
         set_name?: string;
         mask_position?: MaskPosition;
         file_size?: number;
+        file_unique_id?: string;
```

Comment:
Make `file_unique_id` required? It is not optional in the [docs](https://core.telegram.org/bots/api#sticker).